Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -338,7 +338,7 @@ def bump_formula_pr
 
         if reply.to_s.include? "username:"
           formula.path.atomic_write(backup_file) unless ARGV.dry_run?
-          odie "You need to configure hub"
+          odie "Please authentify with hub (eg. by typing 'cd $(brew --repo) && hub issue') and try again."
```

Comment:
How about: `configure hub by running 'hub -C $(brew --repo)' and try again.`

Reply:
That's not enough AFAIK. If you temporarily remove `~/.config/hub` and run that command it works fine.

Reply:
It's a shame hub doesn't have a `hub configure` command.

I think @MikeMcQuaid means
```
hub -C $(brew --repo) issue
```

Reply:
> It's a shame hub doesn't have a hub configure command.

Aye, I went looking for a `hub whoami` command or something but drew a blank.

> hub -C $(brew --repo) issue

👍 That would work.